### PR TITLE
Fixes #18081: move toml data into rudderlang stdlib

### PR DESCRIPTION
**Pull request by gpoblon**
https://issues.rudder.io/issues/18081

### CODE REVIEW

File: rudder-lang/tests/helpers/cis.rl [hunk 1]
```diff
@@ -8,15 +8,16 @@
 ]
 
 resource CIS_redhat7___Enable_Service(service)
+
 CIS_redhat7___Enable_Service state technique() {
   p0="skip_item_${report_data.canonified_directive_id}"
   p1="node.properties[skip][${report_data.directive_id}]"
   @component = "condition_from_variable_existence"
-  condition(p0).from_variable_existence(p1) as condition_from_variable_existence_skip_item___report_data_canonified_directive_id_
+  'condition' at resourcelib.rl:8:10(p0).'from_variable_existence' at resourcelib.rl:61:17(p1) as condition_from_variable_existence_skip_item___report_data_canonified_directive_id_
```
Comment (amousset) [inline rudder-lang/tests/helpers/cis.rl:16]: doesn't look right

**Comment by gpoblon** [comment]
PR updated with a new commit

**Comment by gpoblon** [comment]
Commit modified

**Comment by gpoblon** [comment]
OK, squash merging this PR

**Pull request merged**